crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -226,22 +231,29 @@ impl<'flox> Publish<'flox, NixAnalysis> {
         self.analysis.deref()
     }
 
+    pub async fn build(&self) -> Result<(), PublishError> {
+        let nix = self.flox.nix(Default::default());
+        let command = Build {
+            installables: [self.installable()].into(),
```

Comment:
I think current publish builds all outputs:
https://github.com/flox/flox/blob/eff9f4f0a726cb5ffa632f2672f0577d75becb78/flox-bash/lib/commands/publish.sh#L470

Are we deliberately changing that behavior? Can you add a comment with some explanation of what we've decided?

Reply:
Yeah we should just use `^*` everywhere (also rather than checking individual store paths i think)

Reply:
time to add that to runix

Reply:
flox/runix#47